docs/Operators.md

```diff
@@ -4107,6 +4106,47 @@ expect(node, inputs=[data], outputs=[data],
 </details>
 
 
+### <a name="If"></a><a name="if">**If**</a>
+
+  If conditional
+
+#### Version
+
+This version of the operator has been available since version 1 of the default ONNX operator set.
+
+#### Attributes
+
+<dl>
+<dt><tt>else_branch</tt> : graph (required)</dt>
+<dd>Graph to run if condition is false. Has N outputs: values you wish to be live-out to the enclosing scope. The number of outputs must match the number of outputs in the then_branch.</dd>
+<dt><tt>then_branch</tt> : graph (required)</dt>
+<dd>Graph to run if condition is true. Has N outputs: values you wish to be live-out to the enclosing scope. The number of outputs must match the number of outputs in the else_branch.</dd>
+</dl>
+
+#### Inputs
+
+<dl>
+<dt><tt>cond</tt> : B</dt>
+<dd>Condition for the if</dd>
+</dl>
+
+#### Outputs (1 - &#8734;)
```

Comment:
Does both statements need to return the same Outputs, and types?

Reply:
Yep, it sounds as we discussed.

Reply:
Updated the doc